src/table.ts

```diff
@@ -321,6 +325,14 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
     }
 
     const userStream = through.obj();
+    const end = userStream.end.bind(userStream);
+    userStream.end = () => {
```

Comment:
@callmehiphop what do you think about overriding `end()` here, and cancelling the request stream? (`abort()` maps to the grpc stream's `cancel()`). Is there a better way?

Note that we show in the sample documentation that we will end a stream early when the user calls `this.end()` from a data event handler.

Reply:
Makes sense to me.

I feel like we should have a discussion about making a breaking change to move away from `end()`. IMO its a little confusing because `end()` is a writable method that indicates no more data should be written to the stream. I think if we went all in on destroy instead we could just implement a `_destroy` method that would cancel the request when called. That or something custom like `cancel` or `abort`. 🤷‍♂

Reply:
Yep, abort or cancel are ideal names to me. Although taking out the naming collision, "end()" makes sense, too. I guess I will keep end for now and open an issue about a switch in the future?

Reply:
SGTM!